ssn/index.html

```diff
@@ -465,29 +465,47 @@ <h3>Overview of Classes and Properties</h3>
         </p>
         <figure>
           <img alt="SOSA ontology modules - Observation" src="images/SOSA-OntStructure-Observation.png">
-          <figcaption>Overview of the SOSA classes and properties (observation perspective)</figcaption>
+          <figcaption>Overview of the SOSA classes and properties (observation perspective)
+            <br>TODO Add UltimateFoI
```

Comment:
Is there an issue (or issues) for these TODO annotations? If not I suggest creating it/them. It would then be worthy to cross-reference them here.

Reply:
Just making sure not to overlook them when we re-draw all the figures

Reply:
> Besides the comments by line, is it possible to have predicate names in the figure (ssn/images/SOSA-Collections.png)?

All figures will need to be redrawn. 
This will be done when the ontology is finalized.